scripts/customer_scenarios.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+from functools import cache
+
+import click
+import requests
+import testimony
+
+from robottelo.config import settings
+
+
+@click.group()
+def main():
+    pass
+
+
+def make_path_list(path_list):
+    targets = ['tests/foreman', 'tests/upgrades']
+    if path_list:
+        paths = path_list.split(',')
+        paths = [path for path in paths if any(target in path for target in targets)]
+        return set(paths)
+    else:
+        return targets
+
+
+def get_bz_data(paths):
+    testcases = testimony.get_testcases(paths)
+    result = {}
+    for path, tests in testcases.items():
+        path_result = []
+        for test in tests:
+            test_dict = test.to_dict()
+            test_data = {**test_dict['tokens'], **test_dict['invalid-tokens']}
+            if 'bz' in test_data.keys():
+                if (
+                    'customerscenario' not in test_data.keys()
+                    or test_data['customerscenario'] == 'false'
+                ):
+                    path_result.append([test.name, test_data['bz']])
+        if path_result:
+            result[path] = path_result
+    return result
+
+
+@cache
+def get_response(bzs):
+    bz_fields = [
+        'id',
+        'summary',
+        'status',
+        'resolution',
+        'cf_last_closed',
+        'external_bugs',
+    ]
+    bz_url = settings.bugzilla.url
+    bz_key = settings.bugzilla.api_key
```

Comment:
Need not to initialize the vars as you have used it only once, you can direct use it :)

Non Blocking comment!